assets/js/index.js

```diff
@@ -276,19 +276,40 @@ document.addEventListener('DOMContentLoaded', () => {
     modal.innerHTML = modalData;
 
     // Open modal
-    modal.style.display = 'block';
+    modal.style.display = "block";
 
     // Close modal
-    const closeModal = document.getElementsByClassName('modal-close-icon')[0];
-    closeModal.addEventListener('click', () => {
-      modal.style.display = 'none';
+    const closeModal = document.getElementsByClassName("modal-close-icon")[0];
+    closeModal.addEventListener("click", () => {
+      modal.style.display = "none";
     });
   };
 
   // When user clicks out side of the modal, close the modal.
   document.onclick = (e) => {
     if (e.target === modal) {
-      modal.style.display = 'none';
+      modal.style.display = "none";
     }
   };
 });
+
+// Form Validation
+const email = document.getElementById("email");
+const submit = document.getElementById("submit");
+const alert = document.getElementById("alert-danger");
+
+const submitForm = (e) => {
+  const regex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  const str = email.value;
```

Comment:
- [ ] You are supposed to check if the email is all lowercase by comparing if the email and it's lowercase converted version are the same, ie: `email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string.

Please add this validation and your project ready to go. Cheers 👍